docs/manual/cert.md

```diff
@@ -1,13 +1,52 @@
 
 !!!note
-    For other nodes the automated TLS pipeline is not provided yet and can be addressed by contributors.
+    Currently, automated installation of a node certificate is implemented only for [Nokia SR Linux](kinds/srl.md).
+
+### CA certificate
+
+When generating CA certificate and key, containerlab can take in the following optional parameters:
+
+* `.settings.certificate-authority.key-size` - the size of the key in bytes, default is 2048
+* `.settings.certificate-authority.validity-duration` - the duration of the certificate. For example: `10m`, `1000h`. Max unit is hour. Default is `8760h` (1 year)
+
+### Node certificates
+
+The decision to generate node certificates is driven by either of the following two parameters:
 
-Apart from automated pipeline for certificate provisioning, containerlab exposes the following commands that can create a CA and node's cert/key:
+1. node kind
+2. `issue` boolean parameter under `node-name.certificate` section.
+
+For SR Linux nodes the `issue` parameter is set to `true` by default. For other node kinds the `issue` parameter is set to `false` by default and can be [overridden](nodes.md#certificate) by the user.
+
+## Simplified CLI for CA and end-node keys generation
+
+Apart automated pipeline for certificate provisioning, containerlab exposes the following commands that can create a CA and node's cert/key:
 
 * [`tools cert ca create`](../cmd/tools/cert/ca/create.md) - creates a Certificate Authority
 * [`tools cert sign`](../cmd/tools/cert/sign.md) - creates certificate/key for a host and signs the certificate with CA
 
-With these two commands users can easily create CA node certificates and secure the transport channel of various protocols. [This lab](https://clabs.netdevops.me/security/gnmitls/) demonstrates how with containerlab's help one can easily create certificates and configure Nokia SR OS to use it for secured gNMI communication.
\ No newline at end of file
+With these two commands users can easily create CA node certificates and secure the transport channel of various protocols. [This lab](https://clabs.netdevops.me/security/gnmitls/) demonstrates how with containerlab's help one can easily create certificates and configure Nokia SR OS to use it for secured gNMI communication.
+
+## External CA
+
+Users who require more control over the certificate generation process can use an existing external CA. Containerlab needs to be provided with the CA certificate and key. The CA certificate and key must be provided via `.settings.certificate-authority.[key]|[cert]` configuration parameters.
```

Comment:
It is actually also possible to provide the key and cert via ENV vars:
`CLAB_CA_CERT_FILE` and `CLAB_CA_KEY_FILE`